backend/data/src/main/resources/db/changelog/schema.xml

```diff
@@ -2113,4 +2113,31 @@
             <column name="last_used_at" type="timestamp"/>
         </addColumn>
     </changeSet>
+    <changeSet author="cynthia (generated)" id="1660576021025-1">
+        <addColumn tableName="user_account">
+            <column name="totp_key" type="bytea"/>
+        </addColumn>
+    </changeSet>
+    <changeSet author="cynthia (generated)" id="1660576021025-2">
+        <addColumn tableName="user_account">
+            <column name="mfa_recovery_codes" type="text[]"/>
+        </addColumn>
+    </changeSet>
+    <changeSet author="cynthia (generated)" id="1661345119881-1">
+        <addColumn tableName="user_account">
+            <column name="account_type" type="varchar(255)"/>
+        </addColumn>
+    </changeSet>
+    <changeSet author="cynthia" id="1661345119881-2">
+        <sql dbms="postgresql">
+            UPDATE user_account SET account_type = 'LOCAL';
+            UPDATE user_account SET account_type = 'THIRD_PARTY' WHERE password IS NULL;
```

Comment:
Not really. There are users in the production DB who are registered via third party and have passwords.

Reply:
They still do have local authentication available, and accounts registered via third party are migrated to "local" status upon a password reset flow. The `THIRD_PARTY` flag is really for accounts which only have the OAuth flow as an authentication mean, and is used by the frontend to show or not account security settings

Reply:
Ideally, the OAuth account used could be moved to a separate table of linked accounts which could be used later on for 3rd party integrations, so that it becomes an alternative login flow and also does allow for more than 1 account to be linked.

Reply:
OK